### PR TITLE
fix(): replaced v1 docs references with v2 docs references

### PR DESCRIPTION
**Pull request by brene**
*Issue #, if available:*
N/A

*Description of changes:*
- Replaces all docs v1 links to docs v2 link

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: how-to-write-a-transformer.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<meta http-equiv="refresh" content="0; URL='https://aws-amplify.github.io/docs/js/graphql'" />
+<meta http-equiv="refresh" content="0; URL='https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/js'" />
```
Comment (attilah) [inline how-to-write-a-transformer.md:1]: It should be: ```https://docs.amplify.aws/cli/usage/plugin#custom-graphql-transformers```

File: packages/amplify-cli/scripts/post-install.js [hunk 2]
```diff
@@ -18,11 +18,11 @@ console.log(chalk.green('Successfully installed the Amplify CLI'));
 console.log(chalk.green('----------------------------------------'));
 console.log('\n');
 
-console.log(chalk.green('JavaScript Getting Started - https://aws-amplify.github.io/docs/js/start'));
+console.log(chalk.green('JavaScript Getting Started - https://docs.amplify.aws/start'));
```
Comment (attilah) [inline packages/amplify-cli/scripts/post-install.js:21]: Getting started for JS is here: ```https://docs.amplify.aws/start/q/integration/js```, perhaps the text should be changed from "JavaScript" to "Frontend" ?

File: packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ function warnOnAuth(context, map) {
   if (unAuthModelTypes.length) {
     context.print.warning("\nThe following types do not have '@auth' enabled. Consider using @auth with @model");
     context.print.warning(unAuthModelTypes.map(type => `\t - ${type}`).join('\n'));
-    context.print.info('Learn more about @auth here: https://aws-amplify.github.io/docs/cli-toolchain/graphql#auth \n');
+    context.print.info('Learn more about @auth here: https://docs.amplify.aws/cli/graphql-transformer/directives#auth \n');
```
Comment (attilah) [inline packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js:42]: nit: extra space before newline

File: packages/amplify-provider-awscloudformation/src/aws-utils/aws-amplify.js [hunk 4]
```diff
@@ -59,7 +59,7 @@ function printAuthErrorMessage(context) {
   context.print.warning('As of Amplify CLI version 4.0');
   context.print.warning('A cloud project in the Amplify Console will be created to view your resources.');
   context.print.warning('Please update your IAM policy accordingly based on the following doc.');
```
Comment (attilah) [inline packages/amplify-provider-awscloudformation/src/aws-utils/aws-amplify.js:61]: "doc." ? perhaps we could use "documentation" or something else.

File: packages/amplify-cli/scripts/post-install.js (hunk 2 above)
Comment (brene) [reply packages/amplify-cli/scripts/post-install.js:21]: I think this is fine. This will be more explicit. So resolving as "by design" for now.

**Pull request merged**